### DISCUSSION TITLE
Why does the 5MW FOWTs not have nacelle velocity proportional feedback (Fl_Kp) ?

### DISCUSSION THREAD

**Discussion by Melancho7ee** [NatLabRockies/ROSCO#320] [Announcements]
Dear All,

I am trying to use ROSCO for 5MW ITIBarge FOWT. 

However, I found `Fl_Kp=0` in `DISCON.IN` of 5MW model. Why do not apply nacelle velocity proportional feedback to 5MW FOWTs?

**Comment by dzalkind** [comment]
We have the NREL-5MW model set up for fixed-bottom applications.

To apply floating feedback, please review [this example](https://github.com/NREL/ROSCO/blob/main/Examples/24_floating_feedback.py).  Also, you may want to disable constant power control for floating turbines.  `VS_ConstTq` is the DISCON and tuning yaml input.

Since this isn't necessarily an issue with the software, I'll convert this to a discussion post.
